cogs/economy.py

```diff
@@ -281,22 +235,31 @@ def __init__(self, argument):
 
 
 class Economy:
-    """Economy
+    """Economy - Blue variant
```

Comment:
"Blue variant" has to be removed in order to get this mainstreamed for other bots